src/Content.tsx

```diff
@@ -53,17 +60,24 @@ const handleExportContextMenuClick = (event: MouseEvent) => {
       )
   );
 
+  // Ignore this element on pdf download
+  menuContainer?.setAttribute('data-html2canvas-ignore', 'true');
```

Comment:
The `menuContainer` above this is ugly af (my fault 😅 ). 

You can add something like this in the helper file

```javascript
export const findNearestElementById = (element: Element | null | undefined, targetId: string): Element | null => {
  if (!element) {
    return null;
  }

  // Search for siblings with the target ID
  const sibling = element?.parentElement?.querySelector(`#${targetId}`);
  if (sibling) {
    return sibling;
  }

  // Continue searching up the DOM tree
  return findNearestElementById(element.parentElement, targetId);
}
```

and then use it like this

```javascript
const menuContainer = findNearestElementById(target, MENU_CONTAINER);
```

Reply:
Thank you